src/get-or-create-render-context.js

```diff
@@ -18,18 +18,29 @@ class CustomResourceLoader extends jsdom.ResourceLoader {
     }
 }
 
-const runInContext = function(jsdomContext, fnOrText) {
-    const script = typeof fnOrText === "function"
-        ? "(" + fnOrText.toString() + ")()"
-        : fnOrText;
-
-    if (typeof script !== "string") {
-        throw new Error("Must be a function or text");
+const getScript = function(fnOrText, options) {
+    switch (typeof fnOrText) {
+        case "function":
+            const script = "(\n" + fnOrText.toString() + "\n)()";
+            const _options = Object.assign({}, options);
+            _options.lineOffset = -1;
```

Comment:
I'd rather keep column offsets the same than line offsets, so I moved this the content to the second line and added a one line offset. Don't think it's worth doing a more robus version here. This is really really unlikely to change.

Reply:
Hmm, interesting; I entered that comment with VSCode PRs. I think it dropped some words/characters.